_posts/2021-08-29-mosaic-fusion.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: Mosaic Image Fusion
+author: Volker Hilsenstein, Marvin Albert, and Genevieve Buckley
+tags: [life science]
+theme: twitter
+---
+{% include JB/setup %}
+
+## Executive Summary
+
+This blogpost shows a case study, where a researcher uses Dask for mosaic image fusion - stitching multiple smaller images together seamlessly into a very large field of view. Full code examples are available on GitHub from the `DaskFusion` repository: https://github.com/VolkerH/DaskFusion
+
+This work was done by Volker Hilsenstein, in conjunction with Marvin Albert. Genevieve Buckley wrote the blogpost.
+
+Volker Hilsenstein is a scientific software developer at EMBL in Theodore Alexandrov's lab.
```

Comment:
We need to write about who provided the data here too.

Reply:
Yes, I was actually thinking I might take some images of something nicer specifically for this purpose or some calibration slide.

Reply:
I am slightly against this idea, mostly because I know you are quite busy and I don't want this to fall into the trap described by the saying "don't let perfect be the enemy of good" (i.e. don't try to make things perfect and never quite finish, better to put slightly less good things out more frequently).
You'll have a much better idea of how much extra effort that would actually be, though